src/libraries/react-calendar/components/miniature/CalendarSmall.styled.ts

```diff
@@ -0,0 +1,108 @@
+import { styled } from '@mui/system'
+import Grid from '@mui/material/Grid'
+import Typography, { TypographyProps } from '@mui/material/Typography'
+import { lightBlue } from '@mui/material/colors'
+
+export const Section = styled('section')({
+  minHeight: 264,
+  minWidth: 240,
+  background: 'white',
+})
+
+export const Title = styled(Typography)({
+  marginLeft: 8,
+  textTransform: 'capitalize',
+})
+
+export const Navigation = styled(Grid)({
+  marginRight: 4,
+})
+
+export const DayHeader = styled(Typography)({
+  textAlign: 'center',
+  fontSize: 12,
+  color: lightBlue[800],
+  lineHeight: '26px',
+  padding: 2,
+  borderColor: 'white',
+  borderStyle: 'solid',
+  textTransform: 'capitalize',
+  background: 'white',
+  height: 34.3,
+  width: 34.3,
+})
+
+interface StyledDayProps extends TypographyProps {
```

Comment:
Usually we don't type interface inside styled Component, you should write it inside `<>` right away, the reason for this is that you won't use styled component as distinct component, in most cases it belongs to the main component, in this case to the `CalendarSmall`